ops/ccs-ops-misc/load_test/v1/baseline_tests.py

```diff
@@ -0,0 +1,131 @@
+import random
+from typing import Dict
+import urllib3
+import common.config as config
+import common.data as data
+import common.errors as errors
+import common.test_setup as setup
+from common.url_path import create_url_path
+import common.validation as validation
+from locust import HttpUser, task, events, tag
+
+client_cert = setup.getClientCert()
+server_public_key = setup.loadServerPublicKey()
+setup.disable_no_cert_warnings(server_public_key, urllib3)
+setup.set_locust_env(config.load())
+
+mbis = data.load_mbis()
+eob_ids = data.load_bene_ids()
+last_updated = data.get_last_updated()
+cursor_list = data.load_cursors("v1")
+
+class BFDUser(HttpUser):
+    def on_start(self):
+        copied_eob_ids = eob_ids.copy()
+        random.shuffle(copied_eob_ids)
+
+        copied_mbis = mbis.copy()
+        random.shuffle(copied_mbis)
+
+        copied_cursor_list = cursor_list.copy()
+        random.shuffle(copied_cursor_list)
+
+        self.eob_ids = copied_eob_ids
+        self.mbis = copied_mbis
+        self.cursor_list = copied_cursor_list
+
+    def get_eob(self) -> int:
+      if len(self.eob_ids) == 0:
+            errors.no_data_stop_test(self)
```

Comment:
This is a good point; for instance we have many fewer contract data pieces than generic bene ids; this means the test may end earlier than we want if we equally weight all tests.

Reply:
Also, I think we probably consume those ids faster for some tests than others. There are like five or six that use bene_ids, a couple that use contract ids, and a couple more with MBIs. It may also depend on how quickly the requests come back, since we're firing all of them as quickly as we can.

Reply:
This is a good point, and I didn't really consider that the list of contract URLs is less than the list of `bene_id`s, so in a prolonged test run of the suite we may run out of contract URLs to test against. As it stands, I was seeing approximately 25 requests per second per concurrent user for V2 and about double that for V1, so to your point @andrew-kramer over a 3 minute run we end up only sending about 7500 to 10000 requests _total_.

Note that each Locust `User` _copies_ the initial list of each type of input data (MBIs, `bene_id`s and contract URLs) in its `on_start(...)`. This means each concurrent user has their own set of independent data, by the way.

Reply:
Important note: in my own testing, if you run a Locust Task and it never makes a request to the API (i.e. you return immediately or after discovering that there's no data), then it doesn't count that run in the statistics. So it would be ok to do this:

```python
@task
def some_test(self):
    if self.bene_ids:
        # There is some data in self.bene_ids, so run the test
        self.client.get("url", params={ "_id": self.bene_ids.pop() })
    else:
        # There is no data, so just return and this test won't count
        return
```

If there is a bene_id there, it will run and count the test. If not, then the `if` will fail and the test will return without calling `client.get`. Locust will run other tests for which there is data until it runs out of that, too.